tasks/otel_agent.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from invoke import task
+
+from tasks.libs.common.utils import REPO_PATH, bin_name
+
+BIN_DIR = os.path.join(".", "bin", "otel-agent")
+BIN_PATH = os.path.join(BIN_DIR, bin_name("otel-agent"))
+
+
+@task
+def build(ctx):
+    """
+    Build the otel agent
+    """
+
+    if os.path.exists(BIN_PATH):
+        os.remove(BIN_PATH)
+
+    env = {"GO111MODULE": "on"}
+    build_tags = ['otlp']
+
+    cmd = 'go build -tags="{go_build_tags}" '
+    cmd += '-o {agent_bin} {REPO_PATH}/cmd/otel-agent'
```

Comment:
```suggestion
    cmd = f"go build -tags=\"{' '.join(build_tags)}\" -o {BIN_PATH} {REPO_PATH}/cmd/otel-agent"
```
I'm not sure of the reason why you split this. And maybe for readability I'd suggest using f-strings

Reply:
I'd say it's a copy-paste from the [agent build task](https://github.com/DataDog/datadog-agent/blob/610e0ce8837a8512dda19f54c21913bf666e718f/tasks/agent.py#L198-L218), which is using this form (and hasn't been updated in years).

Reply:
> I'd say it's a copy-paste from the [agent build task](https://github.com/DataDog/datadog-agent/blob/610e0ce8837a8512dda19f54c21913bf666e718f/tasks/agent.py#L198-L218), which is using this form (and hasn't been updated in years).

That's correct. Pushed a commit to update it here as recommended.